src/Microsoft.AspNet.OData.Shared/Formatter/Serialization/SelectExpandNode.cs

```diff
@@ -195,6 +207,7 @@ private void BuildExpansions(SelectExpandClause selectExpandClause, HashSet<IEdm
                     if (allNavigationProperties.Contains(navigationProperty))
                     {
                         ExpandedNavigationProperties.Add(navigationProperty, expandItem.SelectAndExpand);
```

Comment:
Can we remove this line? Getting ExpandedNavigationProperties is going to return a dictionary over ExpandedNavigationPropertiesWithExpandedItem, right?